docs/src/appendixes/object_stores.md

```diff
@@ -155,7 +155,7 @@ spec:
 ## Azure Blob Storage
 
 [Azure Blob Storage](https://azure.microsoft.com/en-us/services/storage/blobs/) is the
-obect storage service provided by Microsoft.
+object storage service provided by Microsoft.
```

Comment:
I wonder how this passes the spellchecker

Reply:
we did not include all md under docs in spell check

Reply:
changed the rule